### PR TITLE
Removed section explaining importing Content ISOs

### PR DESCRIPTION
**Pull request by dubewarsagar**
Removed the section explaining Importing content ISOs. 
The steps and configurations described there are related to Pulp 2 
and only applicable till Satellite 6.9.
The same has been reported and confirmed in the follwing BZ comment.

https://bugzilla.redhat.com/show_bug.cgi?id=2128714#c1

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: guides/doc-Managing_Content/master.adoc
Comment (maximiliankolb) [inline guides/doc-Managing_Content/master.adoc:1]: If you want to remove content, then please delete it.
If you want to hide content for a specific build target, please use conditionals.

Please do not add comments but instead reword your git commit message to let other know what has changed and, more importantly, why.

Regarding cherry-picks: for simplicitiy, we generally do not "skip/jump" Foreman versions.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/doc-Managing_Content/master.adoc [hunk 1]
```diff
@@ -55,9 +55,6 @@ ifdef::satellite,katello[]
 include::common/assembly_importing-kickstart-repositories.adoc[leveloffset=+1]
 endif::[]
 
-[appendix]
-include::common/modules/proc_importing-content-isos-into-a-connected-foreman.adoc[leveloffset=+1]
```
Comment (maximiliankolb) [inline guides/doc-Managing_Content/master.adoc:59]: Please also delete the file itself and check that all anchors within this file are not used anywhere throughout the docs.

**Review by maximiliankolb — COMMENTED** [review]
In am unsure if this only applies to Satellite or also to Katello.

File: guides/doc-Managing_Content/master.adoc (hunk 1 above)
Comment (dubewarsagar) [reply guides/doc-Managing_Content/master.adoc:59]: Not sure why build is failing. I have checked and there is no anchor which has used this as a reference.
Any idea what I am missing here @maximiliankolb ?

Comment (maximiliankolb) [reply guides/doc-Managing_Content/master.adoc:59]: This was most likely just a temporary network fluke. I've restarted the failed GHA.

Comment (dubewarsagar) [reply guides/doc-Managing_Content/master.adoc:59]: It is still failing!

**Review by adamlazik1 — APPROVED** [review]
Summarizing the bugzilla:
> The steps and configurations described there are related to Pulp 2 and only applicable till Satellite 6.9.

Maybe it can be added to commit message as an explanation.

Other than that, LGTM

**Comment by adamlazik1** [comment]
@maximiliankolb any final notes?

**Comment by maximiliankolb** [comment]
@adamlazik1 Works for me!

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   df5a2afcc..73445ebf1  3.1 -> 3.1
   ba0c37831..3eb55913d  3.2 -> 3.2
   5c443d4cf..0ecc2d41a  3.3 -> 3.3
   6d0be8d02..1f60d088e  3.4 -> 3.4
   e531e7af6..529b850e6  3.5 -> 3.5
   dd6b9effa..afa73c61f  3.6 -> 3.6

**Pull request merged**